config/500-controller.yaml

```diff
@@ -66,6 +66,8 @@ spec:
           value: config-logging
         - name: CONFIG_OBSERVABILITY_NAME
           value: config-observability
+        - name: INFORMER_LABEL_SELECTORS_FILTER
+          value: knative.dev/used-by=true
```

Comment:
@nak3 I think it would be more appropriate to disable this by default and let the user decide (would require docs/release notes too). wdyth? If so I will create a PR at the S-O to set it on demand.